rcl/src/rcl/logging.c

```diff
@@ -21,6 +21,7 @@ extern "C"
 #include <inttypes.h>
 #include <stdint.h>
 #include <string.h>
+#include <pthread.h>
```

Comment:
This won't work on windows.

Reply:
my bad, that was my out of radar. got it.